### DISCUSSION TITLE
Inline image with links

### DISCUSSION THREAD

**Discussion by Nevinia210** [pnp/pnpcore#744] [Q&A] [answered]
Hello Everyone,

is it possible to create inline image with link?
in UI there is option for that
![image](https://user-images.githubusercontent.com/42325625/152201217-56c3a92d-07cd-4dd3-8af2-be9f135f53de.png)

Thank you in advance for your answers

**Comment by jansenbe** [comment, marked as answer]
@Nevinia210 : currently is not possible, but I'll move this discussion to our issue list so we can track the ask.

**Reply by jansenbe** [reply]
@Nevinia210 : This has been implemented, check nightly version 1.5.104 or higher
